
# HG changeset patch
# User Daniel Atallah <datallah@pidgin.im>
# Date 1362972661 14400
# Node ID 944ec96bb1037a24435fcdc487f9568197ff5b1d
# Parent  12fe7c5bfccf80e145470a5a8a4018c53e75b3d8
Patch from Evangelos Foutras to link to CyrusSASL from the IRC plugin when needed. Fixes #15517

diff --git a/libpurple/protocols/irc/Makefile.am b/libpurple/protocols/irc/Makefile.am
--- a/libpurple/protocols/irc/Makefile.am
+++ b/libpurple/protocols/irc/Makefile.am
@@ -27,7 +27,7 @@
 st =
 pkg_LTLIBRARIES   = libirc.la
 libirc_la_SOURCES = $(IRCSOURCES)
-libirc_la_LIBADD  = $(GLIB_LIBS)
+libirc_la_LIBADD  = $(GLIB_LIBS) $(SASL_LIBS)
 
 endif
 

